src/StanfordBehat/DrupalExtension/Context/SwsContext.php

```diff
@@ -248,5 +248,14 @@ public function theResponseHeaderShouldNotHave($arg1) {
 
   }
 
-
+  /**
+   * @Given /^the "([^"]*)" module is uninstalled/
+   */
+  public function theModuleIsUninstalled($arg1) {
+    $command = "pm-uninstall -y " . $arg1;
+    $result = $this->getDriver("drush")->$command();
+    if (preg_match('/\[error\]/', $result)) {
```

Comment:
check to make sure running this when the module is already uninstalled doesn't give an error result.

Reply:
Good catch; it (strangely) returns `[success]`.

Reply:
well i guess its not wrong haha.